validator/src/main.rs

```diff
@@ -308,8 +309,20 @@ fn download_ledger(
     Ok(())
 }
 
+const IDENTITY_KEYPAIR_NAME: &str = "identity";
+const STORAGE_KEYPAIR_NAME: &str = "storage-keypair";
+const VOTING_KEYPAIR_NAME: &str = "voting-keypair";
+
 #[allow(clippy::cognitive_complexity)]
 pub fn main() {
+    solana_logger::setup_with_filter(
```

Comment:
😢 I've been trying to inhibit logging setup until after clap argument parsing, to keep all argument error messages on boring old stderr.   What was the need to move this?

Reply:
😢 sorry! No need, `eprintln` is better